tests/test_classifier_old.py

```diff
@@ -18,7 +18,7 @@
 import unittest
 
 from tests.base import VisualTestCase
-from yellowbrick.classifier import *
+from yellowbrick.classifier_old import *
```

Comment:
So it looks like it's just the original file; will you remove this before merging it in?